docs/host-powershell/sample-dotnet2.0-powershell-crossplatform/MyApp/MyApp.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you add a readme.md for this sample

Reply:
make sure it's covered by the markdown tests.

Reply:
Don't add a new readme.md. Instead, add a new section to https://github.com/PowerShell/PowerShell/blob/master/docs/host-powershell/README.md

The new section should be the first section, called `PowerShell Core v6.0.1.1 and Later`

Reply:
Added new section.